game/unicode.cc

```diff
@@ -36,10 +36,11 @@ Converter& UnicodeUtil::getConverter(std::string const& s) {
 	return m_converters.try_emplace(s, Converter(s)).first->second;
 }
 
-std::string UnicodeUtil::getCharset (std::string_view str) {
+std::string UnicodeUtil::getCharset (std::string_view& str) {
```

Comment:
Why (non const) reference here? Is str modified?

Reply:
It can be modified by removeUTF8BOM